webapp/portlet/src/main/webapp/vue-apps/people-list/components/PeopleToolbar.vue

```diff
@@ -103,6 +103,9 @@ export default {
     advancedFilterCountDisplay() {
       return this.advancedFilterCount > 0 ? `(${this.advancedFilterCount})`:'';
     },
+    advancedFilterButtonLabel() {
```

Comment:
Why do we need to have this computed property ?

Reply:
There's no need for a computed property when we have a static label. It can be displayed using interpolation! PR updated @ahamdi